cleanSweStore.pl

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/perl -w
+##
+use Getopt::Long;
+use Pod::Usage;
+
+use strict;
+use warnings;
+
+=pod
+
+=head1 NAME
+
+cleanSweStore.pl - clean swestore from old projects
+
+=head1 SYNOPSIS
+
+ cleanSweStore.pl -help|-man
+ cleanSweStore.pl -projects <file with list of projects> [-execute]
+
+=head1 OPTIONS
+
+=over 4
+
+=item -h|-help
+
+prints out a brief help text.
+
+=item -m|-man
+
+Opens the manpage.
+
+=item -runfolder
+
+Path to a file containing all projects that should be removed from SweStore.
+
+Format, runfolder and projectID seperated by tab: runfoldername1	projectId1
```

Comment:
Changed the text to clarify format. It would have been nice with an example, but POD doesn't handle newlines and that makes it hard to create a good example.